src/Symfony/Bridge/Doctrine/Attribute/MapEntity.php

```diff
@@ -21,13 +21,49 @@
 class MapEntity extends ValueResolver
 {
     public function __construct(
+        /**
+         * The entity class.
+         */
         public ?string $class = null,
+
+        /**
+         * Specify the object manager used to retrieve the entity.
+         */
         public ?string $objectManager = null,
+
+        /**
+         * An expression to fetch the entity. The following variables are available:
+         * - `repository` will be your entity repository.
+         * - Any route wildcards are available as variables.
```

Comment:
request attributes, not route wildcards (which is not even a name used in Symfony anywhere)